website/docs/docs/stubs.md

```diff
@@ -52,13 +53,32 @@ For example, assuming you wanted to use the latest version of a particular contr
 https://<yourdomain>.pactflow.io/pacts/provider/:provider/consumer/:consumer/latest/stub
 ```
 
+## Configuration
+
+You can configure the behaviour of the stub service at runtime, using HTTP headers sent with the stub requests.
+
+| Header | Type | Description | Default |
+|--------|------|-------------|---------|
+| `pactflow-stub-cors` | boolean  | Automatically respond to OPTIONS requests and return default CORS headers | `true` |
+| `pactflow-stub-cors-referer` | boolean | Set the CORS Access-Control-Allow-Origin header to the Referer | true |
+| `pactflow-stub-provider-state` | string | Provider state regular expression to filter the responses by | |
+| `pactflow-stub-provider-state-header-name` | string | Name of the header parameter containing the provider state to be used in case multiple matching interactions are found | |
```

Comment:
I haven't implemented this one, it just uses the pactflow-stub-provider-state header

Reply:
Good point, removed.